core/server/master/src/test/java/alluxio/master/file/FileSystemMasterTest.java

```diff
@@ -918,6 +918,41 @@ public void listStatus() throws Exception {
     }
   }
 
+  @Test
+  public void listStatusRecursive() throws Exception {
```

Comment:
is it easy to add tests with `Permissions`? test recursive listing honors permissions for nested directories

Reply:
done